Sources/GithubAPI.swift

```diff
@@ -200,19 +243,32 @@ public class GithubAPI {
 
 // API Headers
 extension GithubAPI {
-  class func githubAPIHTTPHeaders() -> [String: String] {
+  func githubAPIHTTPHeaders() -> [String: String] {
+    let userAgent = ConfigManager.shared?.configDict["USER_AGENT"] as? String ?? DefaultConfigParams.userAgent
     var headers = [String: String]()
-    headers["Authorization"] = "token \(GithubAuth.accessToken)"
-    LogFile.debug("the access token is: \(GithubAuth.accessToken)")
+    headers["Authorization"] = "token \(self.accessToken)"
+    LogFile.debug("the access token is: \(self.accessToken)")
     headers["Accept"] = "application/vnd.github.machine-man-preview+json"
```

Comment:
Does this scale for other beta API tokens? E.g. the Projects API requires a different Accept header.

Reply:
ah, just found out about this today! I have better customization for this in the upcoming PR due to this, thanks for pointing it out :)